pkg/catalog/helm/helm.go

```diff
@@ -174,65 +214,173 @@ func FetchTgz(url, username, password string) ([]v3.File, error) {
 	return files, nil
 }
 
-func FetchFiles(version *ChartVersion, urls []string, username, password string) ([]v3.File, error) {
-	if len(urls) == 0 {
-		return nil, nil
+func (h *Helm) FetchLocalFiles(version *ChartVersion) ([]v3.File, error) {
+	err := h.lockAndVerifyCachePath()
+	defer h.unlock()
+	if err != nil {
+		return nil, err
+	}
+
+	if len(version.LocalFiles) == 0 && len(version.URLs) == 0 {
+		return nil, errors.New("No files or urls provided for helm fetch")
 	}
 
 	var files []v3.File
-	for _, url := range urls {
-		if strings.HasPrefix(url, "file://") {
-			newFile, err := LoadFile(version, strings.TrimPrefix(url, "file://"))
-			if err != nil {
-				return nil, err
-			}
-			files = append(files, *newFile)
-			continue
+	for _, file := range version.LocalFiles {
+		newFile, err := h.loadFile(version, file)
+		if err != nil {
+			logrus.Errorf("Error loading helm file! [%s]: %s", file, err)
```

Comment:
I think in general errors should be returned and handled on the top level, instead of printing in here.